backend/src/main/resources/db/migration/V1__.sql

```diff
@@ -18,6 +18,29 @@ CREATE TABLE locations
     CONSTRAINT pk_locations PRIMARY KEY (id)
 );
 
+CREATE TABLE roles (
+  id BIGSERIAL PRIMARY KEY,
+  name VARCHAR(50) NOT NULL UNIQUE
+);
+
+
+CREATE TABLE users (
+   id UUID PRIMARY KEY,
+   email VARCHAR(255) NOT NULL UNIQUE,
```

Comment:
You should name the UNIQUE constraint as that will be useful, otherwise postgres will generate a random name.